draft-irtf-cfrg-opaque.md

```diff
@@ -1514,8 +1455,7 @@ Note that if the same pair of user identity IdU and value alpha is received
 twice by the server, the response needs to be the same in both cases (since
 this would be the case for real users).
 For protection against this attack, one would apply the encryption function in
-the construction of EnvU ({{SecEnvU}}) to all the key material in EnvU,
-namely, AOenv will be empty.
+the construction of EnvU to all the key material in EnvU, namely, aad will be empty.
```

Comment:
I guess that aad should be replaced here with cleartext_credentials?

Reply:
Yep, good catch!